tests/pw/utils/helpers.ts

```
@@ -453,7 +453,15 @@ export const helpers = {
     // execute wp cli command
     async exeCommandWpcli(command: string, directoryPath = process.cwd()) {
         process.chdir(directoryPath);
-        command = CI ? `npm run wp-env run tests-cli  ${command}` : `cd ${SITE_PATH} && ${command}`;
+        // Prefer local wp-cli when SITE_PATH is provided; otherwise fallback to docker when CI=true
+        if (SITE_PATH && SITE_PATH.length > 0) {
+            command = `cd ${SITE_PATH} && ${command}`;
+        } else if (CI) {
+            command = `npm run wp-env run tests-cli  ${command}`;
+        } else {
+            // As a last resort, try running in current directory
+            command = command;
```

Biome (2.1.2)

[error] command is assigned to itself.

This is where is assigned.

Self assignments have no effect and can be removed.

(lint/correctness/noSelfAssign)